jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -717,6 +717,12 @@
         jenkins-timeout: 170
         timeout: 70
         trigger-job: 'ci-kubernetes-build'
+    - kubernetes-e2e-gce-gci-qa-docker13:
```

Comment:
Suggest s/gce-gci-qa-docker13/cos-docker-validation/
For 1, qa doesn't make sense here, and 2, we might want to reuse the set of jobs for other docker versions so lets leave the name version-neutral.